mmdet/models/losses/iou_loss.py

```diff
@@ -237,6 +237,79 @@ def ciou_loss(pred, target, eps=1e-7):
     return loss
 
 
+@mmcv.jit(derivate=True, coderize=True)
+@weighted_loss
+def alphaiou_loss(pred, target, alpha=3, eps=1e-7, mode='iou'):
+    # iou mode
+    mode = mode.lower()
+    assert mode in ('iou', 'ciou', 'giou', 'diou')
+
+    # overlap
+    lt = torch.max(pred[:, :2], target[:, :2])
+    rb = torch.min(pred[:, 2:], target[:, 2:])
+    wh = (rb - lt).clamp(min=0)
+    overlap = wh[:, 0] * wh[:, 1]
+
+    # union
+    ap = (pred[:, 2] - pred[:, 0]) * (pred[:, 3] - pred[:, 1])
+    ag = (target[:, 2] - target[:, 0]) * (target[:, 3] - target[:, 1])
+    union = ap + ag - overlap + eps
+
+    # change conventional iou to alpha pow
+    ious = torch.pow(overlap / union + eps, alpha)
+
+    # calculate alpha-iou according mode
+    if mode == 'iou':
+        loss = 1 - ious
+        return loss
+    else:
+        # enclose area
+        enclose_x1y1 = torch.min(pred[:, :2], target[:, :2])
```

Comment:
The `else` is unnecessary.

Reply:
Sorry for not replying you in time, something has bothered me over the two weeks. 
I will resume to work from now, thanks.

Reply:
It's no problem. You can manage progress based on your schedule.